tasks/unscramble-words.md

```diff
@@ -0,0 +1,41 @@
+# Summary
+> Groups had two minutes to unscramble as many words as possible on a list of 24 randomly scrambled words. Each item on the list had only one correct answer, and groups received one point for each correct answer.
+
+# References
+> Engel D, Woolley AW, Jing LX, Chabris CF, Malone TW (2014) Reading the Mind in the Eyes or Reading between the Lines? Theory of Mind Predicts Collective Intelligence Equally Well Online and Face-To-Face. PLOS ONE 9(12): e115212. https://doi.org/10.1371/journal.pone.0115212
+
+# Stimuli
+## The visual components
+From: Warrier V., Bethlehem R.A., Baron-Cohen S. (2017) The “Reading the Mind in the Eyes” Test (RMET). In: Zeigler-Hill V., Shackelford T. (eds) Encyclopedia of Personality and Individual Differences. Springer, Cham. https://doi.org/10.1007/978-3-319-28099-8_549-1
+> Example of stimuli during RME test (not during task itself): 
+> 
+> ![RME Test Question](https://github.com/Watts-Lab/task-mapping/blob/unscramble-words/images/rme%20test.gif)
+
+
+
+
+# Procedure
+## Steps
+> A task flow describing the stages a participant goes through and a formal specification of any conditional or complex relationships therein. 
+> List the steps that make up the task. This could include a training protocol, pre and post surveys, task rounds, etc.  
+> Describe the relationship between the steps. By default they are assumed to be chronological and have no interdependencies, but if they are not chronological, e.g., a choice changes the order of the steps, that should be described in full detail. Similarly, if the steps are interdependent, e.g., performance on one step influences what is seen on another step, this should be described in full detail.
+> 1. Participants were assigned randomly to groups of 4
+> 2. Each of the groups was assigned either face-to-face collaboration or online collaboration. 
+> 3. All participants had personal laptops and worked with their group members in a shared online system to complete the task.
+> 4. Participants in the face-to-face group were seated in a small room with a round table
```

Comment:
Since steps 4/5 are longer descriptions of step two can they be combined and added to step two?